src/Plugins/Hive/Drivers/OpenAi/Prompts/OpenAiImageAltPrompt.php

```
@@ -0,0 +1,83 @@
+use Thinktomorrow\AssetLibrary\Asset;
+use Thinktomorrow\Chief\Forms\Fields\Field;
+use Thinktomorrow\Chief\Plugins\Hive\App\Prompts\HivePrompt;
+use Thinktomorrow\Chief\Plugins\Hive\App\Prompts\Presets\HivePromptDefaults;
+use Thinktomorrow\Chief\Plugins\Hive\Drivers\OpenAiDriver;
+use Thinktomorrow\Chief\Sites\ChiefSites;
+
+class OpenAiImageAltPrompt implements HivePrompt
+{
+    use HivePromptDefaults;
+
+    private string $label = 'Verzin alt tekst voor deze afbeelding';
```

PHPMD (2.15.0)

Avoid unused private fields such as '$label'. (Unused Code Rules)

(UnusedPrivateField)